templates/security/certifications.html

```diff
@@ -3,202 +3,203 @@
+ <div class="u-fixed-width">
+   <h2>Ubuntu security certifications</h2>
+   <p class="u-sv2">We strive to make Ubuntu the platform of choice in regulated and high security environments. Ubuntu Advantage enables access to the certification artifacts as well as the necessary tooling for such environments. The following is a list of the certifications available with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a> on public clouds. Click on each for more detailed information.</p>
+   <table class="p-table--security-certifications">
+     <thead>
+       <th></th>
+       <th class="u-align--center">16.04</th>
+       <th class="u-align--center">18.04</th>
+       <th class="u-align--center">20.04</th>
+     </thead>
+     <tbody>
+       <tr>
+         <td>
+           <a class="/security/fips">FIPS 140-2 Level 1 certification</a>: A US and Canada government cryptographic module certification of compliance with the FIPS140-2 data protection standard. US agencies, their service providers or other institutions that comply with similar requirements (e.g., HIPAA, PCI-DSS) are required to comply with FIPS 140-2.
+         </td>
+         <td class="u-align--center">
+           <div class="u-sv2">
+             <i class="p-icon--success u-sv2">Yes: Tooling and automation</i>
+           </div>
+           <div class="u-sv2">
+             <img src="https://assets.ubuntu.com/v1/8e5cc412-AWS.svg" style="height: 32px; width: 53px;" alt="AWS logo">
+           </div>
+           <div class="u-sv2">
+             <img src="https://assets.ubuntu.com/v1/22fd6473-MicrosoftAzure_logo.svg" style="height: 30px; width: 104px;" alt="Microsoft Azure logo">
+           </div>
+         </td>
+         <td class="u-align--center">
+           <div class="u-sv2">
+             <i class="p-icon--success">Yes: Tooling and automation</i>
+           </div>
+           <div class="u-sv2">
+             <img src="https://assets.ubuntu.com/v1/8e5cc412-AWS.svg" style="height: 32px; width: 53px;" alt="AWS logo">
+           </div>
+           <div class="u-sv2">
+             <img src="https://assets.ubuntu.com/v1/22fd6473-MicrosoftAzure_logo.svg" style="height: 30px; width: 104px;" alt="Microsoft Azure logo">
+           </div>
+         </td>
+         <td class="u-align--center">
+           <small><em>Under<br>validation</em></small>
+         </td>
+       </tr>
+       <tr>
+         <td>
+           <a href="/security/cc">Common Criteria, EAL2, an internationally accepted security certification</a>: A 3rd party attestation of the security mechanisms in the operating system. Ubuntu has a Common Criteria EAL2 certification recognized by CCRA and EU SOGIS members.
+         </td>
+         <td class="u-align--center"><i class="p-icon--success">Yes</i></td>
+         <td class="u-align--center"><i class="p-icon--success">Yes</i></td>
+         <td class="u-align--center">
+           <small><em>Under<br>validation</em></small>
```

Comment:
Common Criteria on 20.04 is not under validation, we should either put a dash or leave it empty, whatever looks better.